pkg/pxc/backup/job.go

```diff
@@ -190,7 +190,61 @@ func (Backup) SetStoragePVC(job *batchv1.JobSpec, cr *api.PerconaXtraDBCluster,
 	return nil
 }
 
-func (Backup) SetStorageS3(job *batchv1.JobSpec, cr *api.PerconaXtraDBCluster, s3 api.BackupStorageS3Spec, destination string) error {
+func SetStorageAzure(job *batchv1.JobSpec, cr *api.PerconaXtraDBClusterBackup) error {
+	azure := new(api.BackupStorageAzureSpec)
+	if cr.Status.Storage != nil && cr.Status.Storage.Azure != nil {
+		azure = cr.Status.Storage.Azure
+	}
+	storageAccount := corev1.EnvVar{
+		Name: "AZURE_STORAGE_ACCOUNT",
```

Comment:
In psmdb operator, we use AZURE_STORAGE_ACCOUNT_NAME/AZURE_STORAGE_ACCOUNT_KEY. Can we use the same to be consistent? 
```

./pkg/psmdb/backup/pbm.go:	AzureStorageAccountNameSecretKey = "AZURE_STORAGE_ACCOUNT_NAME"
./pkg/psmdb/backup/pbm.go:	AzureStorageAccountKeySecretKey  = "AZURE_STORAGE_ACCOUNT_KEY"
```